plugin/plugin.go

```diff
@@ -55,25 +70,29 @@ func (p *plugin) Validate(ctx context.Context, req *validator.Request) error {
 		// check if user is privilged to promote to any env
 		if stringInSlice(req.Build.Trigger, p.privilegedUsers) {
 			logrus.Debugf(
```

Comment:
Use fields so the log is more machine parseable. We should actually switch to zap but that's ok for now. Ex: https://github.com/getgoing/drone-amazon-ssm/blob/master/plugin/plugin.go#L56